autocertkit/network_tests.py

```diff
@@ -515,6 +515,9 @@ def _run_test(self, session, mode):
 
         log.debug("Starting test...")
         results = []
+        # First time try ping to make sure network stable
+        ping(vm1_ip, vm2_bondnic_ip, vm2_bondnic_dev)
```

Comment:
There is a duplicate code, you might bind a variable to the return of ping, and then just results.append(var)

Reply:
It is not a duplicate code, it is just a step to try to ping first but not add into results, you can see later ping serveral times.
```
# First time try ping to make sure network stable
        ping(vm1_ip, vm2_bondnic_ip, vm2_bondnic_dev)

        # Test healthy bond
        results.append(ping(vm1_ip, vm2_bondnic_ip, vm2_bondnic_dev))

        # Test degraded bond
        set_nic_device_status(session, self.piface, 'down')
        results.append(ping(vm1_ip, vm2_bondnic_ip, vm2_bondnic_dev))

        # Test degraded bond
        set_nic_device_status(session, self.piface, 'up')
        set_nic_device_status(session, self.siface, 'down')
        results.append(ping(vm1_ip, vm2_bondnic_ip, vm2_bondnic_dev))
```